spec/rubocop/cop/rails/action_filter_spec.rb

```diff
@@ -41,58 +41,6 @@
     end
   end
 
-  context 'Rails <= 4.0', :rails3 do
-    subject(:cop) { described_class.new(config) }
-
-    let(:cop_config) { { 'Include' => nil } }
-
-    context 'when using action methods' do
-      described_class::FILTER_METHODS.each do |method|
-        it "does not register an offense for #{method}" do
-          expect_no_offenses("#{method} :name")
-        end
-
-        it "does not register an offense for #{method} with block" do
-          expect_no_offenses("#{method} { |controller| something }")
-        end
-      end
-
-      described_class::ACTION_METHODS.each do |method|
-        it "accepts #{method}" do
-          expect_no_offenses("#{method} :something")
-        end
-      end
-
-      it 'does not auto-correct to preferred method' do
-        new_source = autocorrect_source_file('before_filter :test')
-        expect(new_source).to eq('before_filter :test')
-      end
-    end
-
-    context 'when using filter methods' do
-      described_class::ACTION_METHODS.each do |method|
-        it "does not register an offense for #{method}" do
-          expect_no_offenses("#{method} :name")
-        end
-
-        it "does not register an offense for #{method} with block" do
-          expect_no_offenses("#{method} { |controller| something }")
-        end
-      end
-
-      described_class::FILTER_METHODS.each do |method|
-        it "accepts #{method}" do
-          expect_no_offenses("#{method} :something")
-        end
-      end
-
-      it 'does not auto-correct to preferred method' do
-        new_source = autocorrect_source_file('before_action :test')
-        expect(new_source).to eq('before_action :test')
-      end
-    end
-  end
-
   context 'Rails >= 4.0', :rails4 do
```

Comment:
Could this context be eliminated now?

Reply:
Sure! Thanks for your review. I updated it.